### PR TITLE
Adding Ulster Scots translations for NI Household, Individual and static content

### PR DESCRIPTION
**Pull request by berroar**
What is the context of this PR?

I have downloaded the latest Ulster Scots .po files for NI from Crowdin and have added them to the repo. I have also updated the translate script so that the NI Ulster Scots schemas are being generated during the build process.

How to review

Build the the new schemas and then check that the NI household and individual census surveys have been translated to Ulster Scots.

**Note**

Translations for static content are not currently at 100%

I had to remove translations for single letter strings in order for the census household schema to build. The single letter strings are erroneous and need to be removed from crowdin.

### CODE REVIEW

**Comment by berroar** [comment]
[Download Ulster Scots PO files (S)](https://trello.com/c/tXQ3t51e/3308-download-ulster-scots-po-files-s)

**Review by rmccar — APPROVED** [review]
Looks ok to me

**Comment by berroar** [comment]
We have updated the language code for Ulster Scots - we are now using 'eo'. 

We made this change because there were multiple issues being caused by using 'en_US' - it was overriding the default language code 'en' and as a result was causing issues like Ulster Scots appearing for static content when opening the census survey in English.

Will now need to raise a PR to change the language code in the questionnaire launcher.

File: app/setup.py [hunk 1]
```diff
@@ -417,7 +417,7 @@ def setup_secure_cookies(application):
 
 
 def setup_babel(application):
-    application.babel = Babel(application)
+    application.babel = Babel(application, default_locale='en_GB')
```
Comment (ajmaddaford) [inline app/setup.py]: This `default_locale` is no longer needed.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: app/setup.py (hunk 1 above)
Comment (berroar) [reply app/setup.py]: Removed

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -0,0 +1,963 @@
...
+#: templates/static/privacy.html:145
+msgid "The Information Commissioner’s Office (ICO) is the independent organisation tasked with regulating data protection in the UK. The ICO can give you <a href='{url}'>extra information about data protection</a> and your rights and will deal with any complaints you may have about our use of your information."
+msgstr ""
+
+#: templates/static/privacy.html:147
+msgid "You can contact the ICO by:"
+msgstr ""
+
+msgid "October"
+msgstr "OOOOOOOO"
```
Comment (ajmaddaford) [inline app/translations/eo/LC_MESSAGES/messages.po]: This needs to be removed.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**